flow/layers/opacity_layer.cc

```diff
@@ -20,6 +20,12 @@ void OpacityLayer::EnsureSingleChild() {
   }
 
   auto new_child = std::make_shared<flow::TransformLayer>();
+
+  // Be careful: SkMatrix's default constructor doesn't initialize the matrix to
+  // identity. Hence we have to explicitly call SkMatrix::MakeTrans(0, 0).
+  SkMatrix identity = SkMatrix::MakeTrans(0, 0);
```

Comment:
`SkMatrix::setIdentity()` is more idiomatic.

Reply:
Or `SkMatrix::reset()`

Reply:
Ok. I was trying to save one line since `setIndentity` requires two lines:
```
SkMatrix identity;
identity.setIdentity();
```
Will change to `setIdentity()` shortly.